engine/jobmaster/dm/config/config_test.go

```diff
@@ -76,6 +76,14 @@ func TestTaskCfg(t *testing.T) {
 	require.NoError(t, jobCfg.DecodeFile(jobTemplatePath))
 
 	taskCfgs := jobCfg.ToTaskCfgs()
+
+	taskCfgList := make([]*TaskCfg, 0, len(taskCfgs))
+	for _, taskCfg := range taskCfgs {
+		taskCfgList = append(taskCfgList, taskCfg)
+	}
+	jobCfg2 := FromTaskCfgs(taskCfgList)
+	taskCfgs = jobCfg2.ToTaskCfgs()
```

Comment:
what's the purpose of these lines?

Reply:
make sure there is no change after convert jobCfg to taskCfg and convert it back.

Reply:
Do you forget to check jobCfg2 equals jobCfg?